src/WebJobs.Extensions.DurableTask/EndToEndTraceHelper.cs

```diff
@@ -768,19 +841,19 @@ public void ExtensionWarningEvent(string hubName, string functionName, string in
             FunctionType functionType = FunctionType.Orchestrator;
 
             EtwEventSource.Instance.EventGridNotificationCompleted(
-                hubName,
-                LocalAppName,
-                LocalSlotName,
-                functionName,
-                functionState,
-                instanceId,
-                details,
-                (int)statusCode,
-                reason,
-                functionType,
-                ExtensionVersion,
-                IsReplay: false,
-                latencyMs);
+               hubName,
+               LocalAppName,
+               LocalSlotName,
+               functionName,
+               functionState,
+               instanceId,
+               details,
+               (int)statusCode,
+               reason,
+               functionType,
+               ExtensionVersion,
+               IsReplay: false,
+               latencyMs);
```

Comment:
can we un-do this small whitespace diff?

Reply:
ok.